.github/release-please-config.json

```diff
@@ -0,0 +1,15 @@
+{
+  "bootstrap-sha": "0d78723d1f046634bc960a8f550d16fb33815877",
```

Comment:
Added `sha` to this commit - https://github.com/krystal/dennis-client/commit/0d78723d1f046634bc960a8f550d16fb33815877